var/spack/repos/builtin/packages/pandorasdk/package.py

```diff
@@ -29,10 +29,12 @@ class Pandorasdk(CMakePackage):
 
     def cmake_args(self):
         args = [
-            self.define("LC_PANDORA_CONTENT", True),
-            self.define("LAR_PANDORA_CONTENT", True),
             self.define("CMAKE_MODULE_PATH", self.spec["pandorapfa"].prefix.cmakemodules),
-            self.define("CMAKE_CXX_FLAGS", "-std=c++17"),
+            self.define("CMAKE_CXX_FLAGS", "-Wno-error"),
+            self.define(
+                "CMAKE_CXX_STANDARD",
+                self.spec["root"].variants["cxxstd"].value if "root" in self.spec else "17",
```

Comment:
`root` does not seem to be a dependent here.

Reply:
Yes, it's a choice because I think it's very likely ROOT is used with Pandora and then for consistency it would be nice to build the stack with the same standard. There are also other packages from Pandora (like https://github.com/PandoraPFA/LCPandoraAnalysis.git) that do depend on ROOT and those will be built with the standard from ROOT. Not necessary but I think this is the more likely case instead of it being hard coded. And if someone builds these packages standalone then they get the standard that is hard coded.

Reply:
What I mean is that `self.spec["root"]` should not exist ever since `root` is not a package that this depends on.